demo/index.html

```diff
@@ -10,144 +10,78 @@
-        </div>
-      </div>
-
-      <div>
-        <h4>Disabled</h4>
-        <div class="horizontal-section">
-          <paper-icon-button icon="menu" alt="menu" disabled></paper-icon-button>
-          <paper-icon-button icon="favorite" alt="heart" disabled></paper-icon-button>
-          <paper-icon-button icon="arrow-back" alt="arrow-back" disabled></paper-icon-button>
-          <paper-icon-button icon="arrow-forward" alt="arrow-forward" disabled></paper-icon-button>
-          <paper-icon-button icon="clear" alt="clear" disabled></paper-icon-button>
-          <paper-icon-button icon="polymer" alt="polymer" disabled></paper-icon-button>
-          <paper-icon-button src="https://assets-cdn.github.com/images/modules/logos_page/Octocat.png" alt="octocat" disabled></paper-icon-button>
-        </div>
-      </div>
-
-      <div>
-        <h4>Color</h4>
-        <div class="horizontal-section">
-          <paper-icon-button icon="menu" alt="menu" class="blue"></paper-icon-button>
-          <paper-icon-button icon="favorite" alt="heart" class="red"></paper-icon-button>
-          <paper-icon-button icon="arrow-back" alt="arrow-back" class="orange"></paper-icon-button>
-          <paper-icon-button icon="arrow-forward" alt="arrow-forward" class="green"></paper-icon-button>
-          <paper-icon-button icon="clear" alt="clear" class="blue"></paper-icon-button>
-          <paper-icon-button icon="polymer" alt="polymer" class="red"></paper-icon-button>
-          <paper-icon-button class="blue" src="https://assets-cdn.github.com/images/modules/logos_page/Octocat.png" alt="octocat"></paper-icon-button>
-        </div>
-      </div>
-
-      <div>
-        <h4>Size</h4>
-        <div class="horizontal-section">
-          <paper-icon-button icon="favorite" alt="heart" class="huge"></paper-icon-button>
-          <br><br><br>
-          <paper-icon-button icon="polymer" alt="polymer" class="huge"></paper-icon-button>
-        </div>
-      </div>
-
-      <div>
-        <h4>Noink</h4>
-        <div class="horizontal-section">
-          <paper-icon-button noink icon="favorite" alt="heart"></paper-icon-button>
-          <br><br><br>
-          <paper-icon-button noink icon="polymer" alt="polymer"></paper-icon-button>
-        </div>
-      </div>
+          <paper-icon-button disabled icon="reply" title="reply"></paper-icon-button>
+        </template>
+      </demo-snippet>
+
+      <h3>Buttons can hide the ripple effect using the <i>noink</i> attribute</h3>
+      <demo-snippet class="centered-demo">
+        <template>
+          <paper-icon-button noink icon="find-in-page" title="find"></paper-icon-button>
+        </template>
+      </demo-snippet>
+
+      <h3>Buttons can be styled using regular CSS and custom properties</h3>
+      <demo-snippet class="centered-demo">
+        <template>
+          <style is="custom-style">
```

Comment:
I wonder if we should be putting some `link` tags in the templates for the demo snippets so that people know where e.g. `--paper-ping-500` comes from.. WDYT?